zproject/prod_settings_template.py

```diff
@@ -825,6 +825,11 @@
 ## (including hiding upload-related options from UI).
 # MAX_FILE_UPLOAD_SIZE = 100
 
+## Maximum allowed size of uploaded file for realm import, in megabytes.
+## Set MAX_WEB_DATA_IMPORT_SIZE_MB to 0 to disable imports.
+## Set MAX_WEB_DATA_IMPORT_SIZE_MB to None for no limit on upload size.
+# MAX_WEB_DATA_IMPORT_SIZE_MB = None
```

Comment:
I don't think we need this in the template; it's not something most self-hosted systems will want to think about at all.